examples/01-Detailed-Examples/06-cyclic-results.py

```diff
@@ -0,0 +1,74 @@
+"""
+.. _ref_cyclic_results_example:
+
+Extract cyclic results
+======================
+In this script a modal analysis with cyclic symmetry is processed to show
+how to expand the mesh and results.
+"""
+
+###############################################################################
+# Perform required imports
+# ------------------------
+# Perform required imports. # This example uses a supplied file that you can
+# get by importing the DPF ``examples`` package.
+
+from ansys.dpf import post
+from ansys.dpf.post import examples
+
+###############################################################################
+# Get ``Simulation`` object
+# -------------------------
+# Get the ``Simulation`` object that allows access to the result. The ``Simulation``
+# object must be instantiated with the path for the result file. For example,
+# ``"C:/Users/user/my_result.rst"`` on Windows or ``"/home/user/my_result.rst"``
+# on Linux.
+
+example_path = examples.download_multi_stage_cyclic_result()
```

Comment:
@PProfizi this is multistage analysis, it's not really what people call cyclic, I think we might loose them with that

(you can create an example for multistage though)